test/conformance/kuttl/generate/validation/clusterpolicy/target-namespace-scope/01-update-clusterrole.yaml

```diff
@@ -1,20 +1,33 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kyverno:background-controller:additional
+  name: kyverno:background-controller:manage-ns-crossplane-role
   labels:
     app.kubernetes.io/component: background-controller
     app.kubernetes.io/instance: kyverno
     app.kubernetes.io/part-of: kyverno
 rules:
 - apiGroups:
-  - '*'
+  - ""
+  - "iam.aws.crossplane.io"
   resources:
   - namespaces
+  - roles
   verbs:
   - create
   - update
-  - patch
   - delete
   - get
-  - list
\ No newline at end of file
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kyverno:manage-ns-crossplane-role
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kyverno:background-controller:manage-ns-crossplane-role
+subjects:
+- kind: ServiceAccount
+  name: kyverno-background-controller
+  namespace: kyverno
```

Comment:
Why is this CRB here?

Reply:
That was from an old test, there are several other kuttl tests defining CRB. I will send a separate PR to remove all of them, does that work for you?

Reply:
Sure